lib/galaxy/config/sample/datatypes_conf.xml.sample

```diff
@@ -1,6 +1,21 @@
 <?xml version="1.0"?>
 <datatypes>
   <registration converters_path="lib/galaxy/datatypes/converters" display_path="display_applications">
+
+    <!-- Various source code files -->
+    <datatype extension="source.h" type="galaxy.datatypes.data:Text" subclass="true" display_in_upload="true" description="C or cpp header file"/>
+    <datatype extension="source.C" type="galaxy.datatypes.data:Text" subclass="true" display_in_upload="true" description="C source file"/>
```

Comment:
can you use a small `c` here?

Reply:
The files have capital C as extension.

Reply:
Exactly. This is really just a Galaxy identifier.